server/executor/trace_poller.go

```diff
@@ -72,25 +73,42 @@ type tracePoller struct {
 }
 
 type PollingRequest struct {
-	ctx        context.Context
-	test       model.Test
-	run        model.Run
-	count      int
-	hadRequeue bool
```

Comment:
The idea behind changing this object is to make it extendable via headers (just like network protocols work by attaching headers to the packet). Thus, I removed the `context` which can be serialized via context propagation and the `hadRequeue`.

This way, we can have the counter for the selector based polling executor without changing this object and impacting other strategies..